docs/CODE_GUIDELINES.md

```diff
@@ -145,21 +143,21 @@ else
 ```
 
 #### 3.3.2. switch case
-Do not indent `case` and `default`.
+Indent `case` and `default`.
```

Comment:
Is mentioning this explicit required? In the if else and try catch section nothing similar is written?

Reply:
Personally I prefer not indenting case to keep indentation shallow.

Reply:
Maybe we can drop this from documentation, then. However, the current clang-format rules enforce "not indent", which I really do not like.

Reply:
> Personally I prefer not indenting case to keep indentation shallow.

My take: default is indent  and there must be good reasons for exceptions. Saving some line space is imo not a good reason in times were nobody uses 80 lines terminals anymore. ;-)

Reply:
If indentation level gets to high due to many nested switch statements, I bet that the code has other "problems". ;-)

Reply:
Since opinions vary perhaps there is no needs to be explicit about this at all, both styles are equally readable.

Reply:
Will just remove this line, being explicit is it required.